<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStrainStatistics</class>
 <widget class="QDialog" name="DialogStrainStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabStrain">
        <attribute name="title">
         <string>Strain</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="WidgetPlot" name="mainStrainPlot" native="true"/>
         </item>
         <item>
          <widget class="WidgetPlot" name="strainPerPointsPlot" native="true"/>
         </item>
         <item>
          <widget class="WidgetSpectrogram" name="spectrogramPlot" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabStrainRate">
        <attribute name="title">
         <string>Strain rate</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="WidgetPlot" name="mainStrainRatePlot" native="true"/>
         </item>
         <item>
          <widget class="WidgetPlot" name="strainRatePerPointsPlot" native="true"/>
         </item>
         <item>
          <widget class="WidgetSpectrogram" name="spectrogramRatePlot" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="WidgetStrainVideo" name="strainVideoWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPlot</class>
   <extends>QWidget</extends>
   <header>ui/widgetplot.h</header>
   <container>1</container>
   <slots>
    <slot>setIndex(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>WidgetSpectrogram</class>
   <extends>QWidget</extends>
   <header>ui/widgetspectrogram.h</header>
   <container>1</container>
   <slots>
    <slot>setIndex(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>WidgetStrainVideo</class>
   <extends>QWidget</extends>
   <header>ui/widgetstrainvideo.h</header>
   <container>1</container>
   <slots>
    <signal>displayIndexChanged(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>DialogStrainStatistics</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>616</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strainVideoWidget</sender>
   <signal>displayIndexChanged(int)</signal>
   <receiver>mainStrainPlot</receiver>
   <slot>setIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strainVideoWidget</sender>
   <signal>displayIndexChanged(int)</signal>
   <receiver>strainPerPointsPlot</receiver>
   <slot>setIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strainVideoWidget</sender>
   <signal>displayIndexChanged(int)</signal>
   <receiver>spectrogramPlot</receiver>
   <slot>setIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>548</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strainVideoWidget</sender>
   <signal>displayIndexChanged(int)</signal>
   <receiver>mainStrainRatePlot</receiver>
   <slot>setIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strainVideoWidget</sender>
   <signal>displayIndexChanged(int)</signal>
   <receiver>strainRatePerPointsPlot</receiver>
   <slot>setIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strainVideoWidget</sender>
   <signal>displayIndexChanged(int)</signal>
   <receiver>spectrogramRatePlot</receiver>
   <slot>setIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>548</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
